docs/modules/deployment/pages/minion/install.adoc

```diff
@@ -121,9 +121,9 @@ include::message-broker/grpc.adoc[]
 --
 ====
 
-== Check Minion in the Core web UI
+== Check Minion in the core web UI
 
 . Log in to the web UI as an administrative user.
-. Click the gears icon and choose *Manage Minions* in the *Distributed Monitoring* area. 
+. Click the menu:Gears Icon[Distributed Monitoring > Manage Minions].
```

Comment:
```suggestion
. Click the menu:gears icon[Distributed Monitoring > Manage Minions].
```
Will this still work if "gears icon" is lower case? We should capitalize it only if it's a name on the screen.

Reply:
I'm not sure - I saw the syntax elsewhere in our docs. @mmahacek, would this change break the syntax?

Reply:
case is irrelevant.